API/RestSharpAPI/SwaggerJsonApi.csproj

```diff
@@ -20,6 +20,9 @@
     <SpecFlowObsoleteCodeBehindFiles Remove="Drivers\**" />
     <SpecFlowObsoleteCodeBehindFiles Remove="SwaggerDefinitions\**" />
     <SpecFlowObsoleteCodeBehindFiles Remove="TestResults\Logs\**" />
+    <None Update="passwords.json">
```

Comment:
Don't think we need it.  we don't copy passwords.json and this file is used to run tests locally.